lib/wibox/drawable.lua

```diff
@@ -170,7 +170,9 @@ local function find_widgets(_drawable, result, _hierarchy, x, y)
             0, 0, width, height)
         table.insert(result, {
             x = x3, y = y3, width = w3, height = h3,
-            drawable = _drawable, widget = _hierarchy:get_widget()
+            drawable = _drawable, widget = _hierarchy:get_widget(),
+            matrix = _hierarchy:get_matrix_to_parent(),
+            full_path = result,
```

Comment:
Why do you need the full path? What do you even mean by full path here? The `result` table will not be a path. If you have two overlapping widgets, it will e.g. include both, even though neither includes the other.

Do you perhaps want `matrix = _hierarchy:get_matrix_to_device()`?
The matrix to the parent cannot really be useful here, because code using this knows nothing about where the parent widget is. The device (=drawable coordinates) seems a lot more useful.